source/material_model/rheology/diffusion_creep.cc

```diff
@@ -68,17 +69,20 @@ namespace aspect
                                                stress_exponents_diffusion, composition);
             creep_parameters.grain_size_exponent = MaterialModel::MaterialUtilities::phase_average_value(phase_function_values, n_phases_per_composition,
                                                    grain_size_exponents_diffusion, composition);
+	    creep_parameters.water_fugacity_exponent = MaterialModel::MaterialUtilities::phase_average_value(phase_function_values, n_phases_per_composition,
+                                               water_fugacity_exponents_diffusion, composition);
           }
         return creep_parameters;
       }
 
 
 
       template <int dim>
-      double
+      double // Add position to args of this function
```

Comment:
Does this function still need the position argument for this PR? If yes, can you add in the next commit? If not, remove comment?